plugins/cli/plugin.go

```diff
@@ -3,82 +3,62 @@ package cli
 import (
 	"flag"
```

Comment:
This should probably changed to `pflag` as well, so that the provided `printUsage` is displayed.

Reply:
Changed